Gems/Atom/Feature/Common/Code/Include/Atom/Feature/Utils/FrameCaptureBus.h

```diff
@@ -43,45 +45,60 @@ namespace AZ
             //! Capture final screen output for the specified window and save it to given file path.
             //! The image format is determinate by file extension
             //! Current supported formats include ppm and dds. 
-            //! @param filepath The output file path. 
+            //! @param imagePath The output file path. 
             //! @param windowHandle The handle to the AzFrameWork::NativeWindow that is being captured
             //! @return value is the frame capture Id, on failure it will return InvalidFrameCaptureId
-            virtual FrameCaptureId CaptureScreenshotForWindow(const AZStd::string& filePath, AzFramework::NativeWindowHandle windowHandle) = 0;
+            virtual AZ::Outcome<FrameCaptureId, FrameCaptureError> CaptureScreenshotForWindow(const AZStd::string& filePath, AzFramework::NativeWindowHandle windowHandle) = 0;
```

Comment:
nit: It looks like we need to update the param name here to match the doxygen comment (`filePath` -> `imagePath`)